src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -100,8 +100,8 @@ public static IDataView[] LoadFeaturizedSentimentDataset(MLContext mlContext)
             // Download the files
             var dataFiles = DownloadSentimentDataset();
 
-            // Define the columns to load
-            var loader = mlContext.Data.CreateTextLoader(
+            // Define the columns to read
+            var reader = mlContext.Data.CreateTextLoader(
```

Comment:
>reader [](start = 16, length = 6)

nit: if the Name is CreateTextLoader let;s stick with Load.

Reply:
Still reader.

---
In reply to: [261714176](https://github.com/dotnet/machinelearning/pull/2765#discussion_r261714176) [](ancestors = 261714176)